packages/slice-machine/lib/models/common/TrackingEvent.ts

```diff
@@ -12,3 +13,12 @@ export type ReviewTrackingEvent = TrackingEvent & {
   rating: number;
   comment: string;
 };
+
+export type OnboardingTrackingEvent = TrackingEvent & {
+  id: TrackingEventId.ONBOARDING;
+  lastStep: number;
+  maxSteps: number;
+  startTime: number;
+  endTime: number;
+  totalTime: number;
```

Comment:
A question, how is the repository ID found by the CDP project ?
I saw that it's part of the Codec but couldn't figure out how it was sent.

Reply:
those have been removed in favour o more events

Reply:
reop-id is found by the authentication server